README.md

```diff
@@ -1,42 +1,59 @@
+# SC-Web
+## About
+SC-Web is an intelligent knowledge base user interface. Currently, it is a part of [OSTIS Web Platform](https://github.com/ostis-ai/ostis-web-platform). 
+
+The long term idea of this project is to create a universal rendering mechanism for interfaces defined inside knowledge bases.
+
+## Demo
+
+![sc-web-demo](https://github.com/ostis-ai/ostis-project/raw/sc-web-demo/docs/sc-web-demo.gif)
+
+## Quick start
+
+We are providing `ostis/sc-web` Docker image to ease the integration of this UI with existing OSTIS-systems
 
-# Installation
-## Using Docker
-Running image:
 ```sh
-docker run --rm -it -p 8000:8000 ostis/sc-web --server_host=... <any other arguments of app.py>
+# Connect to remote sc-server (or another Docker container)
+docker run --rm -it -p 8000:8000 ostis/sc-web:latest --server_host=<ip or hostname>
+# Connect to server hosted on localhost
+docker run --rm -it --network=host ostis/sc-web:latest
 ```
-### Connecting to sc-server 
-You will need to pass `--server_host=<your hostname>` flag to the container. 
-- In case the `sc-server` is running on another machine or container, simply put its hostname in the template. 
-- If you're running `sc-machine` locally, use [this guide](https://www.howtogeek.com/devops/how-to-connect-to-localhost-within-a-docker-container/) to set up connection to your host's `localhost`. Writing `--server_host=localhost` **will not work**. 
-
-In non-production environment it's easier to pass `--network=host` flag to `docker run` command to connect to `sc-server` running locally.
-## Installing manually
-### Installing requirements:
-* python3
-* pip
-* nodejs
-* npm
-* grunt-cli
-* python modules: tornado, sqlalchemy, numpy, configparser, py-sc-client
-
-You can install deps with
+
+## Installation
+
+  If you're using Ubuntu, you can install dependencies using our script:
+
+  ```sh
+  cd sc-web/scripts
+  ./install_deps_ubuntu.sh
+  pip3 install -r requirements.txt
+  npm install
+  ```
+
+  Otherwise, the following dependencies should be installed:
+
+  - python3
+  - pip
+  - nodejs
+  - npm
+  - grunt-cli
+  - python modules: tornado, sqlalchemy, numpy, configparser, py-sc-client
+
+  ### Building
 
 ```sh
-    pip3 install -r requirements.txt
-    cd ./scripts
-    ./install_deps_ubuntu.sh    #for debian based distros
+npm run build
 ```
 
-### Building
-To install current module you may use internal scripts. Run
+  ### Running
+
+  SC-Web requires `sc-server` to be up and running.
+  
+Use included scripts to launch the server:
 
 ```sh
-    cd scripts
-    ./install.sh
-```
+cd sc-web/scripts
+./run_scweb.sh`
+``` 
 
-### Running
-To run ostis web interface you should build knowledge base and run sc server. You can do it by running `./build_kb.sh` and `./run_sc_server.sh` in [ostis-web-platform](https://github.com/ostis-ai/ostis-web-platform) module. After that you should run `./run_scweb.sh` in */scripts* directory.
+The UI will listen at localhost:8000.
```

Comment:
Should we add some information about configuration?

Reply:
Sounds like a good idea! We should decide which configuration we'll document though, because we have 2 mechanisms for this:
- the `app.py` flags
- the `sc-web.ini` file. 

In case of the former, I do not like the idea of posting the `--help` section of it, though we could note that it's possible to learn more about the script using this flag. In case of the latter - I know literally nothing about this config file, so some suggestions would be appreciated 😄

Reply:
I think it'd be good, but it's not high priority so we could hold this change for a while